velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -1042,8 +1042,12 @@ TEST_F(SharedArbitrationTest, concurrentArbitration) {
       runOrderByTask(vectors, nullptr, numDrivers, pool(), false).data;
   const auto expectedRowNumberResult =
       runRowNumberTask(vectors, nullptr, numDrivers, pool(), false).data;
-  const auto expectedTopNResult =
-      runTopNTask(vectors, nullptr, numDrivers, pool(), false).data;
+  RowVectorPtr expectedTopNResult;
+  {
+    TestScopedAbortInjection testScopedAbortInjection(0);
```

Comment:
Why we need this to get expected result from topN query?

Reply:
Add this to ensure no abort exception if the `abortPct` is 0%